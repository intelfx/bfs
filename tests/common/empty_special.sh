bfs_diff rainbow -empty
