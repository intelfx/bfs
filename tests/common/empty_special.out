rainbow/[1m/[0m
rainbow/exec.sh
rainbow/file.dat
rainbow/file.txt
rainbow/lower.gz
rainbow/lower.tar
rainbow/lower.tar.gz
rainbow/lu.tar.GZ
rainbow/mh1
rainbow/mh2
rainbow/ow
rainbow/sgid
rainbow/sticky
rainbow/sticky_ow
rainbow/sugid
rainbow/suid
rainbow/ul.TAR.gz
rainbow/upper.GZ
rainbow/upper.TAR
rainbow/upper.TAR.GZ
